.eslintrc.js

```diff
@@ -350,7 +350,12 @@ module.exports = {
 			},
 		},
 		{
-			files: [ 'packages/components/src/**/*.[tj]s?(x)' ],
+			files: [
+				// Components package.
+				'packages/components/src/**/*.[tj]s?(x)',
+				// Navigaiton block.
+				'packages/block-library/src/navigation/**/*.[tj]s?(x)',
```

Comment:
👍 I think it's a good call to expand this list as we work through addressing the existing warnings and aim to not introduce new ones.